Greenstand-Token.md

```diff
@@ -584,7 +581,7 @@ Globally ‘tree planting’ is currently being counted as a “pledge” or as
 **TRing**
 
 
-TRing is the native currency of the Greenstand network. They are created by depositing Greenstand Tokens. The number of TRings produced per Greenstand Token is calculated using the Master Relative Value Index. A TRing is divisible by 100 to create Grain.
+TRing is the native currency of the Greenstand network. They are created by depositing Greenstand Tokens. The number of TRings produced per Greenstand Token is calculated using the Master Relative Value Index. A TRing is divisible by 100 to create Seed.
```

Comment:
https://academy.binance.com/en/glossary/wei
In my opinion, we should conform to the existing standard to the best of our ability. I will defer to your judgement, however.

Reply:
Thanks @bstetzer32 This is a good point. 
We should certainly stick with the existing standard, although I believe it should be renamed. 
Do you have any ideas there? - 10^18 is a big number.